gap/examples.gi

```diff
@@ -370,3 +370,43 @@ GeneralisedPetersenGraphCons);
 
 InstallMethod(GeneralisedPetersenGraph, "for integer, integer", [IsInt, IsInt],
 {n, k} -> GeneralisedPetersenGraphCons(IsImmutableDigraph, n, k));
+
+InstallMethod(GeneralisedPetersenGraphCons,
+"for IsImmutableDigraph, integer, int",
+[IsImmutableDigraph, IsInt, IsInt],
+function(filt, n, k)
+  local D;
+  D := MakeImmutable(GeneralisedPetersenGraphCons(IsMutableDigraph, n, k));
+  SetIsMultiDigraph(D, false);
+  SetIsSymmetricDigraph(D, true);
+  return D;
+end);
+
+InstallMethod(StarDigraphCons, "for IsMutableDigraph and one integer",
+[IsMutableDigraph, IsPosInt],
+function(filt, k)
+  local j, graph;
+  graph := EmptyDigraph(IsMutable, k);
+  for j in [1 .. (k - 1)] do
+    DigraphAddEdges(graph, [[j, k], [k, j]]);
+  od;
+  return graph;
+end);
+
+InstallMethod(StarDigraph, "for a function, integer",
+[IsFunction, IsPosInt],
+StarDigraphCons);
+
+InstallMethod(StarDigraph, "for integer", [IsPosInt],
+{k} -> StarDigraphCons(IsImmutableDigraph, k));
+
+InstallMethod(StarDigraphCons,
+"for IsImmutableDigraph, integer",
+[IsImmutableDigraph, IsPosInt],
+function(filt, k)
+  local D;
+  D := MakeImmutable(StarDigraph(IsMutableDigraph, k));
+  SetIsMultiDigraph(D, true);
```

Comment:
This is probably a mistake on my part, since I probably gave a bad explanation. I am under the impression that in directed graphs we consider two vertices to have **multiple edges** between them if there is more than one edge with the same direction between two nodes. I am not 100% sure though. :)